package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wellcometrust/corporate-components",
-  "version": "0.15.11",
+  "version": "0.15.12",
```

Comment:
ohh no! this is after `npm publish --access public` (which failed because of the failed tests). Should I do anything about it? (Manually change it to 0.15.11??)